### DISCUSSION TITLE
I’m getting an SSL error.

### DISCUSSION THREAD

**Discussion by nokemono** [tiangolo/uwsgi-nginx-flask-docker#373] [Questions] [answered]
I’m getting an SSL error.

curl: (35) LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to ***.com:443

What can we do to solve this problem?

**Comment by nokemono** [comment]
It seems that updating Let's Encrypt more than five times a week causes an error.
In rare cases, the packets get jammed, forcing me to restart the container.
Is there any way to prevent ssl from updating more than five times a week?

**Comment by tiangolo** [comment, marked as answer]
TLS certificates would actually be out of scope for this project, I would suggest handling them outside, for example with Traefik, and then it could redirect the traffic to a container using this image.
